src/styles/colour/index.md.njk

```diff
@@ -48,9 +48,14 @@ use `govuk-colour("red")` rather than `$govuk-error-colour`.
         <td class="app-colour-list-column app-colour-list-column--colour">
           <code>{{colour.colour}}</code>
         </td>
+        {% if colour.notes %}
```

Comment:
Tiny bit strange seeing an if/else duplicate the table cell `<td>` but I understand why 😔 

Not something we can fix with `{{-` or `-}}` instead?

```njk
{{- colour.notes -}}
```

Reply:
Ah yeah, I meant to flag this one actually - I originally went with your suggestion, but that generated diffs and I was trying to smoosh the diff as much as possible to make reviewing it simpler.

Happy to do it this way instead though!

Reply:
Not a blocker either way, just stood out from the crowd 😊